timelapse/encoder.py

```diff
@@ -56,6 +56,7 @@ def run(self) -> None:
                     command, capture_output=True, check=True)
             except subprocess.CalledProcessError as e:
                 notify("Timelapse Error:", e.stderr.decode('utf-8'))
+                notify("ffmpeg has not been installed.", not_found_msg)
```

Comment:
Perhaps we can write
```
ffmpeg could not be found.
```

Because perhaps the user has installed `ffmpeg` but we didn't find it. Give the user the benefit of doubt.

Reply:
Good idea. 
The error message has to be short because it will be truncated otherwise. Clicking on it wouldn't help because it would open the script editor (#39).
I'm also worried that the notification could be missed it gets shown right after the first one.
Maybe we can make one notification out of it?

```
notify("Timelapse: ffmpeg not found.", e.stderr.decode('utf-8'))
```
While the error message could look ugly, it might help with troubleshooting; that's why I kept it in.

Reply:
Yes, seems like a fine plan.